app/controllers/schools/dashboard_controller.rb

```diff
@@ -4,5 +4,24 @@ class Schools::DashboardController < Schools::BaseController
   skip_after_action :verify_authorized
   skip_after_action :verify_policy_scoped
 
-  def show; end
+  def show
+    @school = current_user.induction_coordinator_profile.schools.first
+
+    @cohorts = [Cohort.current].map do |cohort|
+      school_cohort = SchoolCohort.find_by(
```

Comment:
This looks unnecessary as well. Once you have a school, school.school_cohorts will give you all the associated cohorts

Reply:
nice!